src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -66,17 +71,69 @@ private boolean validateName(String trustName, Errors errors, String loggingCont
                 && StringValidators.isValidCharacters(trustName, qualifiedFieldName, errors, loggingContext);
     }
 
-    private boolean validateCreationDate(LocalDate dateOfBirth, Errors errors, String loggingContext) {
+    private boolean validateCreationDate(LocalDate creationDate, Errors errors, String loggingContext) {
         String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CREATION_DATE_FIELD);
 
-        return UtilsValidators.isNotNull(dateOfBirth, qualifiedFieldName, errors, loggingContext)
-                && DateValidators.isDateInPast(dateOfBirth, qualifiedFieldName, errors, loggingContext);
+        return UtilsValidators.isNotNull(creationDate, qualifiedFieldName, errors, loggingContext)
+                && DateValidators.isDateInPast(creationDate, qualifiedFieldName, errors, loggingContext);
+    }
+
+    private void validateCeaseDate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+                                   TrustDataDto trustDataDto,
+                                   Errors errors,
+                                   String loggingContext) {
+        final String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        final LocalDate trustCeaseDate = trustDataDto.getCeasedDate();
+
+        if (isTrustStillRequired(overseasEntitySubmissionDto, trustDataDto)) {
+            // Cease date of the trust must be 'null' as there are still Individual and/or Corporate BOs associated with this trust
+
+            if (Objects.nonNull(trustCeaseDate)) {
+                final String errorMessage = ValidationMessages.NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+                setErrorMsgToLocation(errors, qualifiedFieldName, errorMessage);
+                ApiLogger.infoContext(loggingContext, errorMessage);
+            }
+        } else {
+            final LocalDate creationDate = trustDataDto.getCreationDate();
+
+            // Cease date of the trust cannot be 'null' as there are no longer any Individual or Corporate BOs associated with this trust
+
+            if (UtilsValidators.isNotNull(trustCeaseDate, qualifiedFieldName, errors, loggingContext)) {
+                DateValidators.isDateInPast(trustCeaseDate, qualifiedFieldName, errors, loggingContext);
+                DateValidators.isCeasedDateOnOrAfterCreationDate(trustCeaseDate, creationDate, qualifiedFieldName, errors, loggingContext);
```

Comment:
There's no AC for checking if ceased date is on or after creation date, the web doesn't check this either

Reply:
I think AC4 and this line of [this](https://companieshouse.atlassian.net/browse/UAR-1425?focusedCommentId=236545) comment covers it:

**_past date that is before the date the trust was created = invalid_**